contracts/tokens/SecurityToken.sol

```diff
@@ -721,13 +718,10 @@ contract SecurityToken is StandardToken, DetailedERC20, ReentrancyGuard, Registr
     }
 
     function _burn(address _from, uint256 _value, bytes _data) internal returns(bool) {
-        require(_value <= balances[_from], "Value too high");
         bool verified = _updateTransfer(_from, address(0), _value, _data);
         _adjustTotalSupplyCheckpoints();
-        balances[_from] = balances[_from].sub(_value);
-        totalSupply_ = totalSupply_.sub(_value);
+        _burn(_from, _value);
```

Comment:
@maxsam4 @satyamakgec @kostind Should this be super._burn instead? Or otherwise we should maybe rename our _burn function so we don't clash with the OZ namespace.

Reply:
(bit surprised this actually works as currently written)

Reply:
I was confused by this when I first saw it as well. It works because our _burn() takes an extra parameter. Therefore, it's function overloading and not overriding that's happening here.